scripts/upgradable.js

```diff
@@ -52,7 +52,12 @@ async function upgradeUpgradable(wallet, proxyAddress, contractJson, implementat
     return tx;
 }
 
+function getProxy(wallet, proxyAddress) {
```

Comment:
you could move this function to `utils.js`

Reply:
it's specific to the upgradable contract, so i'm preferring to keep it here for now